api/src/main/java/org/openmrs/module/fhir2/api/util/LastnResults.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.fhir2.api.util;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+
+public class LastnResults {
+	
+	String uuid;
+	
+	Date datetime;
+	
+	Map<String, Object> attributes;
+	
+	public LastnResults(Object uuid, Object datetime, Map<String, Object> attributes) {
+		this.uuid = (String) uuid;
+		this.datetime = (Date) datetime;
+		this.attributes = attributes;
+	}
+	
+	public LastnResults(Object[] objects) {
+		uuid = (String) objects[0];
+		datetime = (Date) objects[1];
+	}
+	
+	public Map<String, Object> getAttributes() {
```

Comment:
Instead of explicitly writing getter and setter for each field, it's easier to use the `@Data` annotation from Lombok on the class. It automatically provides us with the getter, setter, toString, hashCode, equals of all the final fields.

Reply:
Even better would be a Java 14 Record class... maybe someday.